admin/base/views.py

```diff
@@ -74,3 +57,15 @@ def form_valid(self, form):
     @property
     def success_url(self):
         raise NotImplementedError
+
+
+class GuidView(DetailView):
+    def __init__(self):
+        self.guid = None
```

Comment:
Why is it necessary to save `guid` as an attribute of the view? I have a slightly bad feeling about overriding `__init__`; we should try to avoid overrides (except for hooks and methods that are meant to be overriden) if possible.